bcbio/variation/vardict.py

```diff
@@ -170,13 +170,17 @@ def _run_vardict_caller(align_bams, items, ref_file, assoc_files,
                 jvm_opts = _get_jvm_opts(items[0], tx_out_file)
                 setup = ("%s && unset JAVA_HOME &&" % utils.get_R_exports())
                 contig_cl = vcfutils.add_contig_to_header_cl(ref_file, tx_out_file)
-                lowfreq_filter = _lowfreq_linear_filter(0, False)
+                use_lowfreq_filter = config["algorithm"].get("use_lowfreq_filter")
+                if use_lowfreq_filter is False:
+                    lowfreq_filter = " | "
+                else:
+                    lowfreq_filter = " | " + _lowfreq_linear_filter(0, False) + " | "
                 cmd = ("{setup}{jvm_opts}{vardict} -G {ref_file} "
                        "-N {sample} -b {bamfile} {opts} "
                        "| teststrandbias.R "
                        "| var2vcf_valid.pl -A -N {sample} -E {var2vcf_opts} "
-                       "| {contig_cl} | bcftools filter -i 'QUAL >= 0' | {lowfreq_filter} "
-                       "| {fix_ambig_ref} | {fix_ambig_alt} | {remove_dup} | {vcfstreamsort} {compress_cmd}")
+                       "| {contig_cl} | bcftools filter -i 'QUAL >= 0' {lowfreq_filter} "
```

Comment:
This is an example of an area where we should try to add the `config_utils.get_program`; it should work for bcftools and hopefully the same for `teststrandbias.R`.